doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -2956,6 +2956,26 @@ The API can also be used to reset the dataset to use the default/inherited value
 
 The default will always be the same provider as for the dataset PID if that provider can generate new PIDs, and will be the PID Provider set for the collection or the global default otherwise.
 
+Reconcile the PID ofa a dataset (If multiple PIDProvider are Enabled)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Dataverse can be configured to use multiple PID Providers (see the :ref:`pids-configuration` section for more information).
+It is possible to update the used PIDProvider of a dataset. This API not not modifies the PIDProvider, but
+reconciles the PID with the current configured provider if it has not already been published.
+This means that the identifier of a previous configured PIDProvider is removed and a new one is created using the current PIDProvider.
+Note that this change does not affect the storage repository where the old identifier is still used.
```

Comment:
(An administrator could move the files manually and set the storagelocationdesignator to false on the entry in the alternativepersistentidentifier table for the old identifier if this is a concern. This is not necessary for Dataverse to function correctly.)